platooning_control/include/pid_controller.h

```diff
@@ -18,6 +18,22 @@
 
 #pragma once
 
+/*
+ * Copyright (C) 2021 LEIDOS.
```

Comment:
This appears to be an invalid merge as there are now two copyrights in this file. This one and the once above.